packages/atomizer/src/rules.js

```diff
@@ -2494,6 +2387,16 @@ module.exports = [
             },
         ],
     },
+    {
+        type: 'pattern',
+        name: 'Outline-color',
+        matcher: 'Oc',
+        allowParamToValue: true,
+        styles: {
+            'outline-color': '$0',
+        },
+        arguments: [colors],
+    },
     {
         type: 'pattern',
         name: 'Outline-offset',
```

Comment:
interesting that this existed without the other rules

Reply:
Ya it must have been an outside contribution